src/routers/incompleteApplication.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express')
+const IncompleteApplication = require('../models/IncompleteApplicationModel')
+const router = new express.Router()
+const authMiddleware = require('../middleware/authMiddleware')
+
+router.put('/editing', authMiddleware, async (req, res) => {
```

Comment:
Same as the comment below, consider the put request route handler name to be something more concise.